common/traffic_light_utils/test/test_traffic_light_utils.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+#include "traffic_light_utils/traffic_light_utils.hpp"
+
+using namespace traffic_light_utils;
```

Comment:
@beginningfan 
Could you fix this `using namespace` to pass cpplint?
It should be something like 
```
namespace traffic_light_utils
{

}  // namespace traffic_light_utils
```

Reply:
Done